css/index.css

```diff
@@ -28,6 +28,28 @@ article, aside, details, figcaption, figure,
 footer, header, hgroup, menu, nav, section {
 	display: block;
 }
```

Comment:
If you contain this area to the same width as the rest of the page and change justify-content: space-around to space-between it will line up the nav links and image to the rest of the page.